### ISSUE TITLE
[Feature request] Linear schedule

### ISSUE THREAD

**Issue by frederikfaye** [MLearing/Keras-DropBlock#1]
Any plans on creating a linear schedule, to linearly decrease `keep_prob` from 1 to the set value during training, as in the paper?

An implementation in PyTorch can be found here: https://github.com/miguelvr/dropblock/blob/master/dropblock/scheduler.py
